css-ecommerce-page/index.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Ecommerce Challenge</title>
+  <link rel="stylesheet" href="styles.css">
+<link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=DM+Sans:wght@400;500&display=swap" rel="stylesheet">
+</head>
+<body>
+  <div class="container">
+    <h1>New Arrivals</h1>
+    <div class="row">
+      <div class="column-one-fifths">
+        <img src="images/dress1.jpeg">
+      </div>
+      <div class="column-one-fifths">
+        <img src="images/ysl-bag.jpeg">
+      </div>
+      <div class="column-one-fifths">
+        <img src="images/key.jpeg">
+      </div>
+      <div class="column-one-fifths">
+        <img src="images/lv-bag.jpg">
+      </div>
+      <div class="column-one-fifths">
+        <img src="images/dress2.jpeg">
+      </div>
+    </div>
+    <div class="row">
+      <div class="column-one-fifths">
+        <span class="bold-text">Free People</span>
+        <span class="gray-text vertical-bar"></span>
+        <span class="font-weight-500 gray-text">&nbsp;Size 6</span>
+        <div class="price">
+          <span class="yellow-bg bold-text">$35</span>
+          <span class="gray-text striked-text font-weight-500">$125 retail</span>
+        </div>
+      </div>
+      <div class="column-one-fifths">
+        <span class="bold-text">Yves Saint Laurent</span>
+        <span class="gray-text vertical-bar"></span>
+        <span class="font-weight-500 gray-text">&nbsp;One-Size</span>
+        <div class="price">
+          <span class="yellow-bg bold-text">$200</span>
+          <span class="gray-text striked-text font-weight-500">$1600 retail</span>
+        </div>
+      </div>
+      <div class="column-one-fifths">
+        <span class="bold-text">Tiffany</span>
+        <span class="gray-text vertical-bar"></span>
+        <span class="font-weight-500 gray-text">&nbsp;One-Size</span>
+        <div class="price">
+          <span class="yellow-bg bold-text">$476</span>
+          <span class="gray-text striked-text font-weight-500">$2000 retail</span>
+        </div>
+      </div>
+      <div class="column-one-fifths">
+        <span class="bold-text">Louis Vuitton</span>
+        <span class="gray-text vertical-bar"></span>
+        <span class="font-weight-500 gray-text">&nbsp;One-Size</span>
+        <div class="price">
+          <span class="yellow-bg bold-text">$95</span>
+          <span class="gray-text striked-text font-weight-500">$1200 retail</span>
+        </div>
+      </div>
+      <div class="column-one-fifths">
+        <span class="bold-text">Reformation</span>
+        <span class="gray-text vertical-bar"></span>
+        <span class="font-weight-500 gray-text">&nbsp;Size 4</span>
```

Comment:
I would put all of these into a `p` element, that may also help you avoid needing a separate element with a massive height to be the vertical line separator. You should be able to make that line with a `border-left` property on the element that holds the item's size.